src/main/java/org/ja13/eau/misc/Coordonate.kt

```diff
@@ -10,6 +9,8 @@ import net.minecraft.util.AxisAlignedBB
 import net.minecraft.util.Vec3
 import net.minecraft.world.World
 import net.minecraftforge.common.DimensionManager
+import org.lwjgl.util.vector.Vector3f
+import javax.vecmath.Vector3d
```

Comment:
Is this really what you meant to return?

Reply:
Vector3d is used in some math, but Vector3f isn't needed.